src/color_picker.coffee

```diff
@@ -205,6 +205,14 @@ Ember.Widgets.ColorPicker = Ember.Component.extend
       ])
     ])
 
+  setupCustomColor: Ember.observer(->
+    selectedColor = @get 'selectedColor'
+    for row in @get('colorRows')
```

Comment:
nit: can you do `return if @get('colorRows').find (row) -> selectedColor in row`